content/blog/20221111_fluentd-v1.15.3-has-been-released.md

```diff
@@ -0,0 +1,69 @@
+# Fluentd v1.15.3 has been released
+
+Hi users!
+
+We have released v1.15.3 on 2022-11-02. ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1153---20221102).
+
+In this release, we fixed several bugs.
+
+## Bug Fixes
+
+### `out_file`: Fix the multi-worker check with `<worker N-M>` directive
+
+In the v1.15.1, we added a new feature to prevent write conflicts between multiple workers.
+
+* [#3808](https://github.com/fluent/fluentd/pull/3808)
+
+However, this fix didn't work correctly with `<worker N-M>` directive.
+
+* [`<worker N-M>` directive](https://docs.fluentd.org/deployment/multi-process-workers#less-than-worker-n-m-greater-than-directive)
+
+With this setting, the configuration check before starting the supervisor mistakenly determines
+that the setting of `out_file` is invalid and causes Fluentd to terminate.
+
+In this release, we fixed this problem.
+
+See [#3942](https://github.com/fluent/fluentd/pull/3942) for more information.
+
+### Fix log rotation on Windows
+
+We can configure log rotation setting as follows.
+
+* [Log Rotation Setting](https://docs.fluentd.org/deployment/logging#log-rotation-setting)
+
+On Windows, the log files must be separated by each process.
+
+> NOTE: When --log-rotate-age is specified on Windows, log files are separated into
+> log-supervisor-0.log, log-0.log, ..., log-N.log where N is generation - 1 due to the system limitation.
+> Windows does not permit delete and rename files simultaneously owned by another process.
+
+However, that log rotation setting in the system-directive didn't satisfy this and
+Fluentd outputed the logs of the supervisor process into the log file of the worker-0 process.
+
+This could cause errors during log rotation.
+
+In this release, we fixed this problem.
+
+See [#3939](https://github.com/fluent/fluentd/pull/3939) for more information.
+
+### Support `!include` directive in YAML config format
+
+In the v1.15.0, we supported YAML config format.
+
+* [Config File Syntax (YAML)](https://docs.fluentd.org/configuration/config-file-yaml)
+* [#3712](https://github.com/fluent/fluentd/pull/3712)
+
+However, `!include` directive supporting was leaked.
+
+In this release, we supported it.
+
+See [#3917](https://github.com/fluent/fluentd/pull/3917) for more information.
+
+## Miscellaneous fixes
+
+* Remove meaningless `oj` options. [#3929](https://github.com/fluent/fluentd/pull/3929)
```

Comment:
It might be more kindful if we mention that it's related with this vulnerability: https://github.com/fluent/fluentd/security/advisories/GHSA-fppq-mj76-fpj2
I don't think it's not serious because probably no one uses this option though.

Reply:
I see, I will add the mention!

Reply:
I fixed this! Thanks for the review!